packages/opentelemetry-plugin-winston/src/types.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Name of params under which the trace information will be saved
+ */
+export const TRACE_PARAM_NAME = 'ot';
+
+/**
+ * Winston Log Chunk Interface
+ */
+export interface WinstonChunk {
```

Comment:
Can you link to documentation for this if possible? I don't know if winston publishes the API for their `write` function, but this interface is very broad.

Reply:
there is no documentation for that, `write` simply accepts just any object which is what WinstonChunk is and then simply split all params into string and put it in a log